src/components/ShareStructure/ShareStructure.vue

```diff
@@ -0,0 +1,1058 @@
+                        <v-list-item-subtitle>
+                          <v-icon>mdi-delete</v-icon> Remove
+                        </v-list-item-subtitle>
+                      </v-list-item>
+                    </v-list>
+                  </v-menu>
+                </span>
+              </div>
+            </td>
+          </tr>
+          <!-- Series Share Edit Form -->
+          <tr
+            v-if="showSeriesEditForm[row.index] && showSeriesEditForm[row.index][index]"
+            :key="`class:${row.index}-Series:${index}-edit-form`"
+          >
+            <td colspan="6">
+              <v-expand-transition>
+                <div class="edit-share-structure-container">
+                  <edit-share-structure
+                    :initialValue="currentShareStructure"
+                    :activeIndex="activeIndex"
+                    :shareId="shareId"
+                    :parentIndex="parentIndex"
+                    :shareClasses="shareClasses"
+                    :resolution-required="resolutionRequired"
+                    @addEditClass="addEditShareClass($event)"
+                    @addEditSeries="addEditShareSeries($event)"
+                    @removeSeries="removeSeries($event, row.index)"
+                    @resolutionPrompt="emitResolutionPrompt($event)"
+                    @resetEvent="resetData()"/>
+                </div>
+              </v-expand-transition>
+            </td>
+          </tr>
+        </template>
+
+        <!-- Series Share Add Form -->
+        <tr v-if="showSeriesAddForm[row.index]">
+          <td colspan="6">
+            <v-expand-transition>
+              <div class="edit-share-structure-container">
+                <edit-share-structure
+                  :initialValue="currentShareStructure"
+                  :activeIndex="activeIndex"
+                  :shareId="shareId"
+                  :parentIndex="parentIndex"
+                  :shareClasses="shareClasses"
+                  :resolution-required="resolutionRequired"
+                  @addEditClass="addEditShareClass($event)"
+                  @addEditSeries="addEditShareSeries($event)"
+                  @removeSeries="removeSeries($event, row.index)"
+                  @resolutionPrompt="emitResolutionPrompt($event)"
+                  @resetEvent="resetData()"/>
+              </div>
+            </v-expand-transition>
+          </td>
+        </tr>
+      </template>
+    </v-data-table>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Emit, Mixins, Prop, Vue, Watch } from 'vue-property-decorator'
+import 'array.prototype.move'
+import { cloneDeep, isEqual, omit } from 'lodash'
+import { v4 as uuidv4 } from 'uuid'
+
+// Components
+import { ActionChip } from '@/components/ActionChip'
+import { ConfirmDialog } from '@/components/ConfirmDialog'
+import EditShareStructure from './EditShareStructure.vue'
```

Comment:
As long as ShareStructure.vue and EditShareStructure.vue are packaged/published together, this is fine/good.